### PR TITLE
gofmt, govet, run those in Travis

### PR DESCRIPTION
**Pull request by broady**
Also small cleanup to cobra_test.

### CODE REVIEW

File: cobra_test.go [hunk 1]
```diff
@@ -916,15 +916,14 @@ func TestRootUnknownCommand(t *testing.T) {
...
 
 	if r.Output != "" {
-		t.Errorf("Unexpected response.\nExpecting to be:\n %q\nGot:\n %q\n", s, r.Output)
+		t.Errorf("Unexpected response.\nExpecting to be: \n\"\"\n Got:\n %q\n", r.Output)
 	}
 
 	r = noRRSetupTestSilenced("--strtwo=a bogus")
 	if r.Output != "" {
-		t.Errorf("Unexpected response.\nExpecting to be:\n %q\nGot:\n %q\n", s, r.Output)
+		t.Errorf("Unexpected response.\nExpecting to be:\n\"\"\nGot:\n %q\n", r.Output)
```
Comment (eparis) [inline cobra_test.go]: This seems to be wrong...  What are you trying to do here?

Comment (broady) [inline cobra_test.go]: Why is it wrong?

The test is checking that `r.Ouput` is `""`.

The message was incorrect before.

Comment (eparis) [inline cobra_test.go]: No, I'm just an idiot.

**Comment by broady** [comment]
Hmmm, @eparis - not sure if the build matrix stuff I just did is correct. I guess we'll see when it hits master.

The PR build seemed to only build for 1.6.3:
https://travis-ci.org/spf13/cobra/builds/154566855

**Pull request merged**